git_workflow (updated).md

```diff
@@ -0,0 +1,59 @@
+The Git Workflow
+==================
+
+## Overview
+
+By now, you should have a solid understanding of git commands, git architecture, and the difference between git and GitHub. It is time to put all of this together, by going through the git workflow.
+
+Imagine, you are at work and your manager wants you to make changes to a repository on GitHub called, *cookies*. Below, are step-by-step instructions on how to do just that.
+
+### Step 1
+
+Once you've located the repository on GitHub, clone it to your local machine. e.g.
+
+	$ git clone <remote repository URL>
+	
+This command will clone the repo from GitHub onto your local git repo.
```

Comment:
It will clone the repo onto your local machine, not your local git repo. That implies that there is an existing repo that you're overwriting. Best to go with 'local machine'.